hermeznetwork/hermez-node
# New value ethTxHash returned in the api when retrieving the batch info

Closes #789

### What does this PR does?

This PR modifies the synchronizer and database to let the node store the ethereum tx hash in the database (batch table). Also, this PR allows the api return the ethereum tx hash when a user retrieves the batch information using the next endpoints:
/v1/full-batches/{{batch_num}}
/v1/batches/{{batch_num}}
/v1/full-batches/

### How to test?

Just run a node in coordinator/sync mode and call this endpoints to check the ethereumTxHash parameter.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
